src/views/ContributionAssistant.vue

```diff
@@ -10,6 +10,9 @@
       <v-tab value="Cleanup" :disabled="!productPriceForms.length">
         3. Cleanup
       </v-tab>
+      <v-tab value="Summary" :disabled="!(productPriceForms.length && (addPricesLoading || productPriceForms.length == numberOfPricesAdded))">
```

Comment:
if the disabled rules are getting too complicated, we can move them to a `method`

Reply:
also, when on the Summary tab, the other tabs should be disabled, no ? to avoid duplicate actions. or do you want to allow users to go back ? add `<v-overlay>` then ?